### PR TITLE
Move notifs to their own file

### PR DESCRIPTION
**Pull request by adamhutchings**

### CODE REVIEW

**Review by JakeRoggenbuck — APPROVED** [review]
I'd recommend dealing with the error or maybe unwrapping

**Review by JakeRoggenbuck — APPROVED** [review]

File: daemon/src/notif.rs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+use notify_rust::Notification;
+
+/**
+ * This is just a thin wrapper around the Notification stuff for now.
+ * The library says it always returns Ok, so we're not worrying about anything
+ * else at the moment.
+ */
```
Comment (Camerooooon) [inline daemon/src/notif.rs:7]: "Javadoc" is rust is denoted with ///

**Review by Camerooooon — CHANGES_REQUESTED** [review]

File: daemon/src/notif.rs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+pub fn notif(summary: &str, msg: &str) {
+    match Notification::new().summary(summary).body(msg).show() {
```
Comment (JakeRoggenbuck) [inline daemon/src/notif.rs:9]: Also, if you want to get the same message but without the empty okay, use the expect method.
```rs
Notification::new().summary(summary).body(msg).show().expect("Unable to display notification!")
```

File: daemon/src/notif.rs (hunk 1 above)
Comment (JakeRoggenbuck) [reply daemon/src/notif.rs:7]: Yea, good catch

**Review by JakeRoggenbuck — APPROVED** [review]

**Review by Camerooooon — APPROVED** [review]

**Pull request merged**